src/GraphQL/GraphQLBuilderExtensions.cs

```diff
@@ -0,0 +1,827 @@
+#nullable enable
+
+using System;
+using System.Linq;
+using System.Reflection;
+using GraphQL.Caching;
+using GraphQL.DI;
+using GraphQL.Execution;
+using GraphQL.Instrumentation;
+using GraphQL.Types;
+using GraphQL.Types.Relay;
+using GraphQL.Utilities;
+using GraphQL.Validation;
+using GraphQL.Validation.Complexity;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Provides extension methods to configure GraphQL.NET services within a dependency injection framework.
+    /// </summary>
+    public static class GraphQLBuilderExtensions
+    {
+        #region - Additional overloads for Register, TryRegister, ConfigureDefaults and Configure -
```

Comment:
I can't find ConfigureDefaults

Reply:
Old code that didn't work out in the end.  Comment needs to be updated

Reply:
#2785